src/cargo/core/resolver/mod.rs

```diff
@@ -71,7 +72,7 @@ mod encode;
 ///
 /// Each instance of `Resolve` also understands the full set of features used
 /// for each package.
-#[derive(PartialEq, Eq, Clone)]
+#[derive(Clone)]
```

Comment:
Let's drop  `derive(Clone)`?  We don't actually clone `Resolve`, and we probably shouldn't because it is a giant pile of `HashMaps`.